src/components/Search/Search.js

```diff
@@ -0,0 +1,12 @@
+import { useState } from 'react';
+
+export default function Search({ setMovieSearch }) {
+  const [search, setSearch] = useState('');
+
+  return (
+    <div>
+      <input value={search} onChange={(e) => setSearch(e.target.value)}></input>
+      <button onClick={() => setMovieSearch(search)}>Search</button>
```

Comment:
Great job with the onClick function to call the API, versus the onChange handler. To make this a solely presentational component, we would want to call our state within the movies view and pass down search and setSearch to this search component.